03week/towersOfHanoi.js

```diff
@@ -13,42 +13,91 @@ let stacks = {
   c: []
 };
 
+/*
+*This function displays the "board" to the user
+*/
 function printStacks() {
   console.log("a: " + stacks.a);
   console.log("b: " + stacks.b);
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
-
+/*
+*This function removes a block from a stack and adds that block to a different stack
```

Comment:
I really like the documentation, and it was obvious you used log statements to debug as you were going along. Nice!